conf/distro/iot2050-debian.conf

```diff
@@ -17,10 +17,14 @@ DISTRO_NAME = "IOT2050 Debian System"
 HOSTNAME ??= "iot2050-debian"
 
 PREFERRED_VERSION_linux-iot2050 ?= "6.1.%"
+PREFERRED_VERSION_linux-iot2050-kbuildtarget ?= "${PREFERRED_VERSION_linux-iot2050}"
 PREFERRED_VERSION_linux-iot2050-rt ?= "6.1.%"
+PREFERRED_VERSION_linux-iot2050-rt-kbuildtarget ?= "${PREFERRED_VERSION_linux-iot2050-rt}"
```

Comment:
Doesn't the missing PREFERRED_VERSION...-native bit you?

Reply:
Should we append the -native to the linux-iot2050? How it looks like when it bites us? At least the recipe parsing has no problem and kernel/rt-kernel is building

Reply:
You will see multiple provider warnings when bitbake identifies them during parsing.

Reply:
Didn't see any warnings... I guess due to isar version? Let me try

Reply:
-kbuildtarget and -native were introduced by the same isar changes. It takes two or more providers for those targets to make bitbake complain. I cannot explain, though, why 6.x-upstream should trigger the warning for kbuildtarget but not for native.

BTW, iot2050-linux-rt should only have one provider at this stage.

So much for the theory...